index.md

```diff
@@ -4,27 +4,30 @@ NASA Astrophysics is developing the Fornax Initiative, a cloud-based system that
 brings together data, software, and computing so that researchers can focus on science.
 
 
-## Tutorial Notebooks
+## User Documentation
```

Comment:
I don't think these changes are correct, with this PR the rendered HTMLs will have the fornax documentation first and then the notebooks.